core/src/main/kotlin/org/evomaster/core/problem/rest/service/SecurityRest.kt

```diff
@@ -303,10 +301,115 @@ class SecurityRest {
             handleStackTraceCheck()
         }
 
+        if (!config.isEnabledFaultCategory(DefinedFaultCategory.SQL_INJECTION)) {
+            LoggingUtil.uniqueUserInfo("Skipping experimental security test for sql injection as disabled in configuration")
```

Comment:
we should remove these `Skipping ` logs, as they are now flooding the logs. if needed, they can be put as `debug` option, and not UserInfo